workshops/tunes/README.md

```diff
@@ -0,0 +1,203 @@
+
+I hope you understand the starter, now you may be getting bored so let's get making!
+
+# Detecting a key press
+
+To play a note in our project we will have the user press a key on their keyboard. Javascript has an event for this: `document.onkeydown`. This is fired when ever a key is pressed down.
+
+For example, in the below snippet:
+
+```javascript
+document.onkeydown = function(e){
+	console.log('Key pressed!')
+}
+```
+
+Whenever a key is pressed it will log 'Key pressed!'. You can try it too! Just type this into you `script.js` file, click 'Run', then click the browser preview and press a key. You should see the logs in the terminal.
+
+The next step is knowing what key we pressed, we want to get a bit more info to do that. We can do this with the following snippet. The snippet gets all the information from the event that fired the function.
+
+```javascript
+document.onkeydown = function(e){
+  e = e || window.event;
+  var key = e.which || e.keyCode;
+  console.log(key)
+}
+```
+
+You'll see that when running this, our key is a number!? That's because each key has it's own code. [keycode.info](https://keycode.info/) is a super handy tool to help you identify the codes for each key.
+
+<img src="https://cloud-7ownn8t7c.vercel.app/screenshot_2020-09-17_at_7.14.26_pm.png" width="380" alt="keycode.info's UI">
+
+Now that we know how to identify the key, let's only log the event if it is the `s` key. It's keycode is `83` We'll use an if/else statement for this.
+
+```javascript
+document.onkeydown = function(e){
+  e = e || window.event;
+  var key = e.which || e.keyCode;
+  if(key===83){console.log('S had been pressed')}
+}
+```
+
+We've done it!
+
+## Playing a sound
+
+Now it's time to play a sound with Tone.js! First let's start up Tone.js, add the following to the top of the Javascript file:
+
+```javascript
+Tone.start();
+const synth = new Tone.Synth().toDestination();
+```
+
+This starts up Tone.js and then creates a synth for us to play a note. 
+
+We're going to do this in a Javascript function, to make our clean. 
+
+```javascript
+function playNote(note) {
+	synth.triggerAttackRelease('C4', "8n");
+}
+```
+
+Here's how we create a function. The name of our function will be `playNote` and it will need a variable called `note` to be provided when the function is called. Then inside of that function, it plays the C4 key for the duration of 8 notes. 
+
+However, we aren't using our `note` variable. Let's use this. We're going to add it in using template strings. Template strings allow us to add variables into our strings. To do this we use the <code>`</code> character instead of  <code>'</code>. 
```

Comment:
I think 'template literals' will be a better word here.

Reply:
imo template strings is better as string is a word they'll be more familiar with

Reply:
like a literal, what's that?

Reply:
I'll put in brackets

Reply:
sure